CoreFoundation/CMakeLists.txt

```diff
@@ -489,20 +489,29 @@ else()
                            -fno-common)
 endif()
 if(CF_DEPLOYMENT_SWIFT)
-  if("${CMAKE_C_SIMULATE_ID}" STREQUAL "MSVC")
-    target_compile_options(CoreFoundation
-                           PRIVATE
-                             /clang:-fcf-runtime-abi=swift)
-    target_compile_options(CFURLSessionInterface
-                           PRIVATE
-                             /clang:-fcf-runtime-abi=swift)
+  if(CMAKE_C_COMPILER_VERSION VERSION_LESS 8.0.0 AND CMAKE_SYSTEM_NAME STREQUAL Android)
```

Comment:
Looks like condition is wrong. Android NDK 19 has clang 8.0.2 but it is not enough

Reply:
Hmm, this could be the NDK folks changing the versioning.  https://github.com/llvm-mirror/clang/blob/release_80/lib/CodeGen/CodeGenModule.cpp indicates that my changes are in the release.

Reply:
I don’t know why the version reports as 8.0.2, but it doesn’t include your changes. They are distributing something based on r339409 according to `--version`, and your changes seemed to happen in r345222 (I miss SVN revision numbers).

Seems that NDKv20 includes "8.0.7" which seems to include your changes (seems like all of them). Maybe we can use `VERSION_LESS 8.0.7` since this only deals with Android.

Reply:
WFM